src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1071,6 +1071,19 @@ private OperatorMemberCrefSyntax ParseOperatorMemberCref()
             return SyntaxFactory.OperatorMemberCref(operatorKeyword, checkedKeyword, operatorToken, parameters);
         }
 
+        private SyntaxToken TryEatCheckedKeyword(bool isConversion)
+        {
+            SyntaxToken checkedKeyword = TryEatToken(SyntaxKind.CheckedKeyword); // PROTOTYPE(CheckedUserDefinedOperators) : consider gracefully recovering from erroneous use of 'unchecked' at this location 
+
+            if (checkedKeyword is not null &&
+                (isConversion || SyntaxFacts.IsAnyOverloadableOperator(CurrentToken.Kind)))
+            {
+                checkedKeyword = CheckFeatureAvailability(checkedKeyword, MessageID.IDS_FeatureCheckedUserDefinedOperators, forceWarning: true);
```

Comment:
> I believe we want to try and defer this to binding, rather than parsing, unless it is significantly simpler to implement in parsing.

These are not errors, but warnings. And I am simply follow the pattern already established in this component.